bots/image-refresh

```diff
@@ -74,7 +75,7 @@ TRIGGERS = {
         "ubuntu-stable"
     ],
     "openshift": [
-        # FIXME: need to test a rhel-7.x branch here, once we can
+        "fedora-30/container-kubernetes@cockpit-project/cockpit/rhel-7.7"
```

Comment:
trailing comma please, and let's run the "regular" rhel-7-7@cockpit-project/cockpit/rhel-7.7 tests as well, as they exercise kubernetes in a different way.

Reply:
Updated and triggered these new tests here to see, if they succeed or fail